src/components/ChannelList.js

```diff
@@ -213,6 +218,17 @@ class ChannelList extends PureComponent {
           refreshing: false,
         };
       });
+
+      // Set a channel as active and move it to the top of the list.
+      if (this.props.customActiveChannel) {
```

Comment:
Can we move this logic to line 202? That way we will only make call to `setActiveChannel` once

Reply:
Yeah, the problem is we don't have the channels in state yet over there.

Reply:
ah ok!!